### PR TITLE
Fixed warnings

### PR DESCRIPTION
**Pull request by RomanPodymov**
Good evening.
Thank you for this library.
In this pull request I just fixed all warnings and made a small refactoring.

### CODE REVIEW

File: Pod/Classes/LocationManager.swift [hunk 1]
```diff
@@ -92,39 +92,41 @@ open class LocationManager: NSObject, CLLocationManagerDelegate {
...
+        }
+    }
+    
+    private func setupRequestPermissionsStrategy(rejection: (Error) -> Void) -> Bool {
+        
+        if askForLocationServicesFulfillments.isEmpty {
+            
             if Bundle.main.object(forInfoDictionaryKey: "NSLocationAlwaysUsageDescription") != nil {
-
+
```
Comment (mkj-is) [inline Pod/Classes/LocationManager.swift:105]: I recommend turning off whitespace-only lines in Xcode. It makes diffs much more cleaner.

**Review by mkj-is — APPROVED** [review]
Thank you, we currently do not use the library and use another methods to get the user location, so this was a bit outdated.

**Comment by RomanPodymov** [comment]
Hello @mkj-is 
This library has 8 forks, probably my changes will be useful for developers who created all these forks.

**Comment by mkj-is** [comment]
That's true. I am merging this PR. Thank you.

File: Pod/Classes/LocationManager.swift (hunk 1 above)
Comment (RomanPodymov) [reply Pod/Classes/LocationManager.swift:105]: @mkj-is Or I can add SwiftLint for this repository.

Comment (RomanPodymov) [reply Pod/Classes/LocationManager.swift:105]: @mkj-is [Swiftlint integration](https://github.com/thefuntasty/LocationManager/pull/9)

**Pull request merged**